build.gradle.kts

```diff
@@ -1,8 +1,17 @@
 /*
- * This file was generated by the Gradle 'init' task.
+ * Copyright 2008-present MongoDB, Inc.
```

Comment:
Nit: change to 2024

Reply:
I might add a copyright section in README.md's IDE config as per https://www.jetbrains.com/help/idea/copyright.html, so we don't need to go through the chore for each file to be created.